### PR TITLE
Feature std dev

### PR DESCRIPTION
**Pull request by brechtfm**
Add daily standard deviation as required by SR1.1.1.

### CODE REVIEW

File: inflammation-analysis.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def main(args):
     for filename in InFiles:
         inflammation_data = models.load_csv(filename)
 
-        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data)}
+        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data), **(models.s_dev(inflammation_data))}
```
Comment (langtonhugh) [inline inflammation-analysis.py]: Testing review comment

File: inflammation/models.py [hunk 2]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```
Comment (langtonhugh) [inline inflammation/models.py]: Could `s_dev` have a clearer (descriptive) name?

File: inflammation/models.py [hunk 3]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
...
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
+    devs = []
+    for entry in data:
+        devs.append((entry - mmm) * (entry - mmm))
+
+    s_dev2 = sum(devs) / len(data)
```
Comment (langtonhugh) [inline inflammation/models.py]: Related to the above, maybe  `s_dev2` could have a descriptive name that differentiates it from `s_dev` without having to read and understand the code?

File: inflammation/models.py (hunk 2 above)
Comment (langtonhugh) [reply inflammation/models.py]:
```suggestion
def std_dev(data):
```

**Review by langtonhugh — COMMENTED** [review]
Here's a test suggestion:

- The std dev function could give an error if there's no data (or non-numerical data),

File: inflammation/models.py (hunk 2 above)
Comment (brechtfm) [reply inflammation/models.py]: Fixed via f34bfc9

File: inflammation/models.py (hunk 3 above)
Comment (brechtfm) [reply inflammation/models.py]: Function no longer uses the confusingly named s_dev2 variable ([9ff5286](https://github.com/brechtfm/python-intermediate-inflammation/commit/9ff5286ef9f5b987c35be7474cf567c219e58582))

**Comment by brechtfm** [comment]
> Here's a test suggestion:
> 
>     * The std dev function could give an error if there's no data (or non-numerical data),

Tests for empty and non numerical data have been added in [9e60035](https://github.com/brechtfm/python-intermediate-inflammation/commit/9e600354eea709ac5dc6784f6b8cb1fa8ad0ab90)

File: inflammation-analysis.py (hunk 1 above)
Comment (brechtfm) [reply inflammation-analysis.py]: Not sure about this comment I updated the function call to include the new clearer function name ([c54d33a](https://github.com/brechtfm/python-intermediate-inflammation/commit/c54d33ac298696072c3fd416fb75fae836a7f2ce))

**Review by langtonhugh — APPROVED** [review]

**Pull request merged**